src/components/StudentGroup/index.tsx

```diff
@@ -31,6 +32,11 @@ interface Props {
 
 const StudentGroup = observer((props: Props) => {
     const [removedIds, setRemovedIds] = React.useState<string[]>([]);
+    const [bulkRemovedIds, setBulkRemovedIds] = React.useState<string[]>([]);
+    const [imported, setImported] = React.useState<{
+        ids: string[];
+        fromGroups: StudentGroupModel[] | undefined;
+    }>();
```

Comment:
Are those states really different, or could we use `imported/setImported` for both usecases?

Reply:
Technically, we could represent bulk-removed ids in the imported state. However, they are semantically different: The former should display a "...have been removed" message with a re-add undo, whereas the latter should display a "...have all been added" message with a remove-undo (hope that makes sense).